lib/rubocop/cop/rspec/rails/have_http_status.rb

```diff
@@ -37,12 +37,14 @@ class HaveHttpStatus < ::RuboCop::Cop::Base
 
           def on_send(node)
             match_status(node) do |response_status, to, match, status|
+              return unless status.to_s.match?(/\A\d+\z/)
```

Comment:
Nitpick: “404 “ with a trailing space would be ignored. Should it?

Reply:
I think that is an acceptable edge case.